pkg/apis/ceph.rook.io/v1/annotations_test.go

```diff
@@ -62,6 +62,7 @@ func TestCephAnnotationsMerge(t *testing.T) {
 		"mgr":            {"mgrkey": "mgrval"},
```

Comment:
update the commit message with a proper description https://rook.io/docs/rook/latest-release/Contributing/development-flow/#commit-structure